apps/docs/content/troubleshooting/upload-file-size-restrictions-Y4wQLT.mdx

```diff
@@ -1,42 +1,40 @@
 ---
 title = "Upload file size restrictions"
 github_url = "https://github.com/orgs/supabase/discussions/27431"
-date_created = "2024-06-20T23:15:48+00:00"
+date_created = "2025-09-03T23:15:48+00:00"
 topics = [ "storage" ]
 keywords = [ "upload", "file", "size", "restriction" ]
 database_id = "3b0aa0f2-9608-4691-94d6-4fca8a32130e"
 ---
 
-You can view the max permissible upload size for your plan in the [docs](https://supabase.com/docs/guides/storage/uploads/file-limits).
+You can view the maximum file upload size the [docs](https://supabase.com/docs/guides/storage/uploads/file-limits). It differs based on your plan and on any bucket or global limits you might have already set.
 
-## There are two ways to control the max upload sizes:
+## Edit the maximum file upload size
 
-The first way is through the [global storage settings](https://supabase.com/dashboard/project/_/storage/settings):
+There are two ways to edit the maximum file upload size:
 
-<img
-  width="920"
-  alt="Screenshot 2024-06-20 at 7 06 57 PM"
-  src="https://github.com/supabase/supabase/assets/91111415/c33acc4a-efd8-4746-ac98-ddc71e17f8f1"
-/>
+- Global file upload size limit
+- Bucket-level file upload size limit
 
-The second way is at the [bucket level](https://supabase.com/dashboard/project/_/storage/buckets/)
+### Global file upload size limit
 
-1. Edit a bucket's configurations:
+To edit the global file upload size limit, change the value of _Global file size limit_ in the top section of [Storage Settings](https://supabase.com/dashboard/project/_/storage/settings).
 
-   <img
-     width="355"
-     alt="Screenshot 2024-06-20 at 7 07 48 PM"
-     src="https://github.com/supabase/supabase/assets/91111415/51719d9f-3644-40ed-9aaa-8b21fff41634"
-   />
+If they are set higher than your new global limit, you’ll be asked to edit or remove any existing bucket-level limits first. The global limit takes precedence.
 
-2. Change the file upload restriction if set:
-   <img
-     width="340"
-     alt="Screenshot 2024-06-20 at 7 07 56 PM"
-     src="https://github.com/supabase/supabase/assets/91111415/917f3cf6-81c4-444f-9ac1-f26eec5eac0c"
-   />
+### Bucket-level file upload size limit
 
-## Different upload methods impose file size restrictions:
+Follow these steps to edit the maximum file upload size at the bucket level:
 
-- [Standard uploads can only transfer up to 5GBs](https://supabase.com/docs/guides/storage/uploads/standard-uploads?queryGroups=language&language=js). However, for files above 6MB, the below methods are more performant and reliable
+1. Open a bucket in the [Storage Buckets](https://supabase.com/dashboard/project/_/storage/buckets/) page.
+2. Tap the overflow (three vertical dots) button, and then tap _Edit bucket_.
```

Comment:
Very minor haha but wondering if we should say "Select" instead of "Tap" (Tap feels like mobile)

Reply:
Thanks. Was trying to get away from a desktop-only action. “Select” is good, will this instead!